c2corg_ui/templates/helpers/edit.html

```diff
@@ -50,9 +50,14 @@ <h3 class="heading show-phone" translate>summary</h3>
       </div>
       % endif
       ## QUALITY
-      <div class="full comment" ng-class="{'has-success': saveCtrl.quality}" ng-init="qualityTypes = ${quality_types}">
-        <label translate>document_quality</label>
-        <select ng-options="type for type in qualityTypes" ng-model="saveCtrl.quality" class="form-control"><option value=""></option></select>
+      <div class="full comment" ng-class="{'has-success': saveCtrl.quality}">
+        <label translate>quality</label>
+        <select ng-model="saveCtrl.quality" class="form-control">
+          <option value=""></option>
+          % for q in quality_types:
```

Comment:
Have you tried to change the scope when compiling the modal template? I would prefer we use the usual way to build and translate a dropdown.